SMLHelper/Patchers/SpritePatcher.cs

```diff
@@ -54,5 +55,48 @@ internal static void Patch()
             Logger.Error("SpritePatcher was unable to find a sprite dictionary");
             return null;
         }
+
+#elif BELOWZERO
+
+        private static Dictionary<SpriteManager.Group, Dictionary<string, UnityEngine.Sprite>> _groups;
+        private static Dictionary<SpriteManager.Group, Dictionary<string, UnityEngine.Sprite>> Groups => _groups ?? (_groups = (Dictionary<SpriteManager.Group, Dictionary<string, UnityEngine.Sprite>>)groupsInfo.GetValue(null));
+
+        private static Dictionary<string, Dictionary<string, UnityEngine.Sprite>> _atlases;
+        private static Dictionary<string, Dictionary<string, UnityEngine.Sprite>> Atlases => _atlases ?? (_atlases = (Dictionary<string, Dictionary<string, UnityEngine.Sprite>>)atlasesInfo.GetValue(null));
```

Comment:
It looks like these two large code blocks are the same in everything except the sprite type.  
Could we use this to consolidate the code back into one single block?
```CS
using System.Collections.Generic;
using System.Reflection;
using Assets;
#if SUBNAUTICA
Sprite = Atlas.Sprite
#elif BELOWZERO
Sprite = UnityEngine.Sprite
#endif

Reply:
https://github.com/SubnauticaModding/SMLHelper/pull/126/commits/e36cd2f20ac8dd44bd4145d11598595ff97d75d8